test/lit.cfg

```diff
@@ -425,7 +425,7 @@ if platform.system() == 'Darwin' and (run_os == 'macosx' or run_os == 'darwin'):
     # <rdar://problem/16678410> Adding more libraries with -lfoo to REPL is broken
     config.available_features.add('swift_repl')
     config.available_features.add('swift_interpreter')
-elif platform.system() == 'Linux':
+elif platform.system() == 'Linux' and not (platform.processor().startswith('armv6') or platform.processor().startswith('armv7')):
```

Comment:
The JIT produces ```R_ARM_REL32```, which are illegal.  Until that is addressed, the interpreter cannot be used on ARM platforms.